dev/playbooks/roles/fedora-common-packages/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: update packages cache
+  yum: update_cache=yes
+  become: yes
```

Comment:
If you call this role from a playbook then become and become_method can be specified once and used everywhere.